tests/monitoring/prometheus_utils.go

```diff
@@ -67,11 +67,11 @@ func getAlerts(cli kubecli.KubevirtClient) ([]prometheusv1.Alert, error) {
 }
 
```

Comment:
@machadovilaca will you be moving this file and the vm_monitoring.go files under the monitoring package?  Can this be done in this PR?

Reply:
nop, this files are from e2e tests, not implementation